tests/baselines/reference/instancePropertyInClassType.errors.txt

```diff
@@ -31,8 +29,8 @@ tests/cases/conformance/classes/members/classTypes/instancePropertyInClassType.t
         r.y = 4;
         var r6 = c.y(); // error
                    ~
-!!! error TS2349: This expression is not callable.
-!!! error TS2349:   Type 'Number' has no call signatures.
+!!! message TS6232: This expression is not callable because it is a 'get' accessor. Did you mean to use it without '()'?
+!!! related TS2728 tests/cases/conformance/classes/members/classTypes/instancePropertyInClassType.ts:4:13: 'y' is declared here.

```

Comment:
I'm not convinced this is valuable. If it is, it should really be on the base "This expression is not callable." error as well. @DanielRosenwasser can you decide on this? I trust your opinion on related spans.